### DISCUSSION TITLE
Stack for advanced web application

### DISCUSSION THREAD

**Discussion by dawosch** [vercel/next.js#27411] [Help]
Hey, i'm working with ReactJS since a few years.
Now i want to create my first web application with NextJS.
But i think i'm to stupid.

Thats my current stack:
- NextJS: 11.0.1
- Typescript: 4.3.5

I created my projekt with the create-next-app with typescript support.
Then i made my first steps which was a great experience:
- Routing out of the box
- Authentication through next-auth to a lot of provider
- ...

But then i started to add a database to my projekt.
And now im stucking.

In other projects (not nextjs) i used for database operations the `typeorm` framework.
But this doesn't seems to work in the nextjs enviroment.

When i define my entities i get an error that i can't use "import" and i should set something to "module" in my package.json.

So i would to know how you add databases to your nextjs app?

I know there are services like firebase (firestore), but i want to use a local database.

**Comment by leerob** [comment]
If you want to use a local database, I would recommend using SQLite with Prisma!

https://www.prisma.io/nextjs
